Automattic/jetpack
# VideoPress: Persist library type

Make sure we persist library type across sessions.

#### Changes proposed in this Pull Request:

* Add library type to `local storage`, and make sure to load it back.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to `Dashboard`.
* Toggle library type to list.
* Reload the page.
* It should load on list mode.
